### DISCUSSION TITLE
Can I use changesets in a non-monorepo project?

### DISCUSSION THREAD

**Discussion by frencojobs** [changesets/changesets#892] [Q&A] [answered]
I'm aware that monorepos are where changesets shine the most at. I've used changesets in my monorepos for that reason but I'd also like to keep my publishing workflow (changesets with actions + bot) consistent in all of my projects. So my question is "does it make sense to use changesets not in a monorepo?".

I've gone ahead and tried it in a project and it seems to work just fine. I just wanted to make sure if there are any caveats that I should be aware of.

**Comment by Andarist** [comment, marked as answer]
Yes, you can totally do this. I'm also using it like this, for example [here](https://github.com/Andarist/react-textarea-autosize)
